standalone/content/css/webapp.css

```diff
@@ -25,6 +26,130 @@ body,
   background: #000;
 }
 
+#main {
+  position: relative;
+}
```

Comment:
This doesn't seem to be necessary; let's get rid of it.  Even if something goes wrong, I don't think this is where we'd want to put the relative.